sklearn/model_selection/_split.py

```diff
@@ -798,24 +817,24 @@ class StratifiedGroupKFold(_BaseKFold):
     >>> X = np.ones((17, 2))
     >>> y = np.array([0, 0, 1, 1, 1, 1, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0])
     >>> groups = np.array([1, 1, 2, 2, 3, 3, 3, 4, 5, 5, 5, 5, 6, 6, 7, 8, 8])
-    >>> cv = StratifiedGroupKFold(n_splits=3)
-    >>> for train_idxs, test_idxs in cv.split(X, y, groups):
-    ...     print("TRAIN:", groups[train_idxs])
-    ...     print("      ", y[train_idxs])
-    ...     print(" TEST:", groups[test_idxs])
-    ...     print("      ", y[test_idxs])
-    TRAIN: [1 1 2 2 4 5 5 5 5 8 8]
-           [0 0 1 1 1 0 0 0 0 0 0]
-     TEST: [3 3 3 6 6 7]
-           [1 1 1 0 0 0]
-    TRAIN: [3 3 3 4 5 5 5 5 6 6 7]
-           [1 1 1 1 0 0 0 0 0 0 0]
-     TEST: [1 1 2 2 8 8]
-           [0 0 1 1 0 0]
-    TRAIN: [1 1 2 2 3 3 3 6 6 7 8 8]
-           [0 0 1 1 1 1 1 0 0 0 0 0]
-     TEST: [4 5 5 5 5]
-           [1 0 0 0 0]
+    >>> sgkf = StratifiedGroupKFold(n_splits=3)
+    >>> sgkf.get_n_splits(X, y)
+    3
+    >>> print(sgkf)
+    StratifiedGroupKFold(n_splits=3, random_state=None, shuffle=False)
+    >>> for i, (train_index, test_index) in enumerate(sgkf.split(X, y, groups)):
+    ...     print(f"Fold {i}:")
+    ...     print(f"  Train: index={train_index}, group={groups[train_index]}")
+    ...     print(f"  Test:  index={test_index}, group={groups[test_index]}")
```

Comment:
What do you think about the following to avoid the line being too long ?
```suggestion
    ...     print(f"Fold {i}:")
    ...     print(f"  Train: index={train_index}")
    ...     print(f"         group={groups[train_index]}")
    ...     print(f"  Test:  index={test_index}")
    ...     print(f"         group={groups[test_index]}")
```